tools/build_pak_files.py

```diff
@@ -84,18 +103,24 @@
         txt_file = os.path.realpath(os.path.join(output_dir, scene_id + "-" + args.version_tag + "-" + platform + ".txt"))
         pak_file = os.path.realpath(os.path.join(output_dir, scene_id + "-" + args.version_tag + "-" + platform + ".pak"))
 
-        perforce_content_scene_dir = os.path.realpath(os.path.join(perforce_content_scenes_dir, scene_id))
-
         # We do not want to use os.path.realpath(...) here, because that will resolve to the directory inside the user's Perforce workspace.
         # Instead, we want this path to refer to the symlinked version inside the user's unreal project directory.
         unreal_project_content_scene_dir = os.path.join(unreal_project_content_dir, "Scenes", scene_id)
 
-        if spear.path_exists(unreal_project_content_scene_dir):
-            print(f"[SPEAR | build_pak_files.py] File or directory or symlink exists, removing: {unreal_project_content_scene_dir}")
-            spear.remove_path(unreal_project_content_scene_dir)
+        if not args.skip_create_symlinks:
+            perforce_content_scene_dir = os.path.realpath(os.path.join(perforce_content_scenes_dir, scene_id))
+
+            if spear.path_exists(unreal_project_content_scene_dir):
+                print(f"[SPEAR | build_pak_files.py] File or directory or symlink exists, removing: {unreal_project_content_scene_dir}")
+                spear.remove_path(unreal_project_content_scene_dir)
+
+            print(f"[SPEAR | build_pak_files.py] Creating symlink: {unreal_project_content_scene_dir} -> {perforce_content_scene_dir}")
+            os.symlink(perforce_content_scene_dir, unreal_project_content_scene_dir)
 
-        print(f"[SPEAR | build_pak_files.py] Creating symlink: {unreal_project_content_scene_dir} -> {perforce_content_scene_dir}")
-        os.symlink(perforce_content_scene_dir, unreal_project_content_scene_dir)
+            # Apart from starter_content_0000, there should only be one other scene.
```

Comment:
Move this assert out of the if statement.

```
        # Now that we have created a symlink, our Unreal project should contain exactly two scenes: starter_content_0000 and scene_id
        ignore_names = [".DS_STORE"]
        unreal_project_scenes = { os.path.basename(x) for x in os.listdir(unreal_project_content_scenes_dir) if x not in ignore_names }
        assert unreal_project_scenes == {"starter_content_0000", scene_id}
```

Reply:
Done